src/controllers/commands/CommandController.cpp

```diff
@@ -1355,6 +1355,106 @@ void CommandController::initialize(Settings &, Paths &paths)
 
         return deleteMessages(channel, messageID);
     });
+
+    this->registerCommand("/unmod", [](const QStringList &words, auto channel) {
+        if (words.size() < 2)
+        {
+            channel->addMessage(makeSystemMessage(
+                "Usage: \"/unmod <username>\" - Revoke moderator status from a "
+                "user. Use \"/mods\" to list the moderators of this channel."));
+            return "";
+        }
+
+        auto currentUser = getApp()->accounts->twitch.getCurrent();
+        if (currentUser->isAnon())
+        {
+            channel->addMessage(
+                makeSystemMessage("You must be logged in to unmod someone!"));
+            return "";
+        }
+
+        auto *twitchChannel = dynamic_cast<TwitchChannel *>(channel.get());
+        if (twitchChannel == nullptr)
+        {
+            channel->addMessage(makeSystemMessage(
+                "The /unmod command only works in Twitch channels"));
+            return "";
+        }
+
+        auto target = words.at(1);
+        stripChannelName(target);
+
+        getHelix()->getUserByName(
+            target,
+            [twitchChannel, channel](const HelixUser &targetUser) {
+                getHelix()->removeChannelModerator(
+                    twitchChannel->roomId(), targetUser.id,
+                    [channel, targetUser] {
+                        channel->addMessage(makeSystemMessage(
+                            QString("You have removed %1 as a moderator of "
+                                    "this channel.")
+                                .arg(targetUser.displayName)));
+                    },
+                    [channel, targetUser](auto error, auto message) {
+                        QString errorMessage =
+                            QString("Failed to remove channel moderator - ");
+
+                        using Error = HelixRemoveChannelModeratorError;
+
+                        switch (error)
+                        {
+                            case Error::UserMissingScope: {
+                                errorMessage += "Missing required scope. "
+                                                "Re-login with your "
+                                                "account and try again.";
+                            }
+                            break;
+
+                            case Error::UserNotAuthorized: {
+                                errorMessage += "you don't have permission to "
```

Comment:
```suggestion
                                errorMessage += "You don't have permission to "
```
same nitpick from /mod